### PR TITLE
Feature/search internal logs button

### PR DESCRIPTION
**Pull request by hsekowski-splunk**
image

### CODE REVIEW

File: src/main/webapp/components/table/TableExpansionRow.jsx [hunk 1]
```diff
@@ -53,7 +54,14 @@ export function getExpansionRow(colSpan, row, moreInfo) {
                         })}
                     </>
                 ) : (
-                    <DL termWidth={250}>{getExpansionRowData(row, moreInfo)}</DL>
+                    <>
+                        <DL termWidth={250}>{getExpansionRowData(row, moreInfo)}</DL>
+                        <Button
+                            to={`/app/search/search?q=search%20index%3D_internal%20source%3D*${row.name}*`}
+                            openInNewContext
+                            label={`Show ${row.name} input events`}
```
Comment (tbalar-splunk) [inline src/main/webapp/components/table/TableExpansionRow.jsx:62]: @hsekowski-splunk I'm not sure how important it is to use internationalization (i18n) for the button label.

Comment (tbalar-splunk) [reply src/main/webapp/components/table/TableExpansionRow.jsx:62]: Also, I will add some CSS later to this button.

Comment (hsekowski-splunk) [reply src/main/webapp/components/table/TableExpansionRow.jsx]: @tbalar-splunk  ,
i18n added

**Review by tbalar-splunk — APPROVED** [review]

**Review by artemrys — APPROVED** [review]

**Pull request merged**